appveyor.yml

```diff
@@ -8,49 +8,32 @@ environment:
     # Avoid long paths on Windows
     STACK_ROOT: "c:\\s"
     STACK_WORK: ".w"
-    CACHE_DIR: "c:\\c"
     WORK_DIR: "c:\\w"
+    CACHE_S3_VERSION: v0.1.1
+    AWS_REGION: us-west-1
+    S3_BUCKET: appveyor-ci-cache              
+    AWS_ACCESS_KEY_ID: 
+      secure: 0OtMCUmJ4V7fNRPvKjzYxSO19I200rPG8WRW/qn1QPk=
```

Comment:
@domenkozar, do we have it documented somewhere, what user we're using for accessing the cache bucket?

Reply:
@deepfire Don't know about the documentation part, but the user is created by terraform here: https://github.com/input-output-hk/iohk-ops/pull/221/files#diff-fefc5952bb2ab240eaa7c2245608fc79R7 and it's default name will be `appveyor-ci-cache-user`

Reply:
Thanks again!